docs/fsharp/using-fsharp-on-azure/queue-storage.md

```diff
@@ -30,67 +30,53 @@ Next, use a [package manager](package-management.md) such as [Paket](https://fsp
 
 Add the following `open` statements to the top of the `queues.fsx` file:
 
-[!code-fsharp[QueueStorage](~/samples/snippets/fsharp/azure/queue-storage.fsx#L1-L3)]
+[!code-fsharp[QueueStorage](../../../samples/snippets/fsharp/azure/queue-storage.fsx#L1-L2)]
```

Comment:
Why were all the paths changed from ~/ to ../../../?

Reply:
Because all the paths ~/ could not link to code page.